src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -28,30 +28,29 @@ export class CalcitePickList {
 
   /**
    * When true, the items will be sortable via drag and drop.
-   * Only applies when mode is configuration
+   * This only applies when mode is configuration.
    */
   @Prop({ reflect: true }) dragEnabled = false;
 
   /**
-   * Mode controls the presentation of the items in their selected and deselected states.
-   * Selection mode shows either radio buttons or checkboxes depending on the value of multiple
-   * Configuration mode relies on a color highlight on the edge of the item for selected
-   * Mode must be set to configuration for drag and drop behavior to work.
+   * This mode controls the presentation of the items in their selected and deselected states.
+   * Selection mode shows either radio buttons or checkboxes depending on the value of 'multiple'.
+   * Configuration mode relies on a color highlight on the edge of the item for the selected.
+   * For drag and drop behavior to work, configuration mode must be set.
    */
   @Prop({ reflect: true }) mode: "selection" | "configuration" = "selection";
 
   /**
-   * Multpile Works similar to standard radio buttons and checkboxes.
+   * Multpile works similar to standard radio buttons and checkboxes.
    * It also affects the presented icon when in Selection mode.
    * When true, a user can select multiple items at a time.
-   * When false, only a single item can be selected at a time,
-   * When false, selecting a new item will deselect any other selected items.
+   * When false, only a single item can be selected at a time and selecting a new item will deselect any other selected items.
    */
   @Prop({ reflect: true }) multiple = false;
 
   /**
-   * The heading label for the entire Pick List.
-   * Not to be confused with the heading for an individual item or for a sub-group of items.
+   * The heading label for the entire Pick List
+   * (not to be confused with the heading for an individual item or for a sub-group of items).
```

Comment:
This should be marked deprecated. Can you double check if you merged in latest from master?